lib/kubernetes-deploy/deploy_task.rb

```diff
@@ -441,6 +441,7 @@ def confirm_cluster_reachable
         end
       end
       raise FatalDeploymentError, "Failed to reach server for #{@context}" unless success
+      KubernetesDeploy::Errors.server_version_warning(server_version, @logger)
```

Comment:
Since the current class here is also namespaced inside `KubernetesDeploy`, you don't need the prefix here